app/change_sets/workflow_note_change_set.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class WorkflowNoteChangeSet < Valkyrie::ChangeSet
+  validates :author, :note, presence: true
```

Comment:
Because it's weird required. You can not fill it out, it just won't make a comment. If you set required to true here it'll make a required label and chrome will prevent the form from submitting without a comment.